web_widget_slick/static/tests/js/web_widget_slick.js

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2017 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+odoo.define_section('web_widget_slick', ['web.core', 'web.form_common'],
+                    function(test) {
```

Comment:
Hrmmmm this tabbing is bugging the crap out of me. I think it would be best to just leave the function declaration on the first line - it won't be that much longer than your other ones and we don't have a 79 char hard limit like we do in Python.